src/v/kafka/requests/schemata/generator.py

```diff
@@ -180,11 +180,11 @@
 }
 
 # a few kafka field types specify an entity type
-entity_type_map = dict(
-    groupId=("kafka::group_id", "string"),
-    topicName=("model::topic", "string"),
-    brokerId=("model::node_id", "int32"),
-)
+entity_type_map = dict(groupId=("kafka::group_id", "string"),
+                       transactionalId=("kafka::transactional_id", "string"),
+                       topicName=("model::topic", "string"),
+                       brokerId=("model::node_id", "int32"),
+                       producerId=("kafka::producer_id", "int64"))
```

Comment:
if you end the parameter list with a comma, does it avoid reformatting the whole thing?

Reply:
got it